libdnf.spec

```diff
@@ -154,6 +155,7 @@ popd
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 %{_datadir}/gir-1.0/Dnf-*.gir
+%{_datadir}/vala/vapi/
```

Comment:
We could merge this and then apply a change to split vala bindings into its own subpackage `libdnf-vala` and have it require the `vala` package.